.github/workflows/release-docker.yml

```
@@ -0,0 +1,45 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
```

I'm pushing a tag to perform a release, so it should be:

```suggestion
on:
  push:
    tags:
      - 'v*'
```